### ISSUE TITLE
Add weight decay regularizer to layers

### ISSUE THREAD

**Issue by mwinton** [mwinton/w266-final-project#71]
This is another component of Yang's model.  It's applied to all layers *except* sentence embeddings.

Yang implements as a weight decay on the cost function:
https://github.com/zcyang/imageqa-san/blob/master/src/san_att_conv_twolayer.py#L120-L132

`options['weight_decay'] = 0.0005`

However, Keras doesn't have a global weight decay.  Instead we have to add a regularizer (kernel and/or bias) to each layer.

This thread seems relevant: https://github.com/keras-team/keras/issues/2717

See https://keras.io/regularizers/
I believe we want `keras.regularizers.l2(0.0005)`
